admin_guide/quota.adoc

```diff
@@ -401,8 +412,8 @@ spec:
 [[create-a-quota]]
 == Creating a Quota
 
-To create a quota, first define the quota to your specifications in a file, for
-example as seen in
+To create a quota, first define the quota to your specifications in a file, 
+for example as seen in
```

Comment:
maybe s/for example as seen in/such as the

Reply:
@kalexand-rh That middle phrase is awkward: _to your specification_ sounds odd.
"To create a quota, first define the quota in a file, such as the examples in..."

Reply:
I think that does sound better @mburke5678!